client/src/Components/DonationCard.jsx

```diff
@@ -166,25 +166,23 @@ export default function DonationCard() {
               className={classes.root}
               autoComplete="off"
             >
-              <Card className={classes.root}>
+              <Card className={classes.root} style={{ display: 'flex', flexDirection: 'column' }}>
                 <CardActionArea>
                   <CardMedia
                     className={classes.media}
                     title={currentDonation && currentDonation.title}
                     image={currentDonation && currentDonation.images[0].url}
                   />
-                  {/* <CardContent>
-                    <Carousel>
+                </CardActionArea>
+                {/* <Carousel>
                       {items.map((item, i) => (
                         <Item
                           className={classes.item}
                           key={i}
                           item={item}
                         />
                       ))}
-                    </Carousel>
-                  </CardContent> */}
-                </CardActionArea>
+                    </Carousel> */}
                 <CardActions className={classes.actions}>
                   <Typography>{currentDonation && currentDonation.donor.username}</Typography>
```

Comment:
I'd factor out all the first checks for currentDonation into a single one at the top that checks whether or not to render the card bottom